frontend/src/views/BackendSessionExpiredView.vue

```diff
@@ -5,24 +5,24 @@
 </template>
 
 <script>
-import { AuthRoutes } from '@/utils/constants'
+import { mapState } from 'pinia'
+
 import { useAuthStore } from '@/stores/auth'
-import { mapActions } from 'pinia'
+import { AuthRoutes } from '@/utils/constants'
+
 export default {
   name: 'BackendSessionExpiredView',
   data() {
     return {
-      link: AuthRoutes.SESSION_EXPIRED,
+      link: '/stuff',
     }
   },
   mounted() {
-    window.location = document.getElementById('logout_href').href
+    const logoutPath = this.isMinistryUser ? AuthRoutes.LOGOUT_IDIR : AuthRoutes.LOGOUT
+    window.location = `${logoutPath}?sessionExpired=true`
   },
-  methods: {
-    ...mapActions(useAuthStore, ['setJwtToken']),
-    logout() {
-      this.setJwtToken()
-    },
+  computed: {
+    ...mapState(useAuthStore, ['isMinistryUser', 'userInfo']),
```

Comment:
userInfo is unused. also, my eslint suggest that computed should be above mounted

Reply:
Updated.